src/frontend/efiling-frontend/.eslintrc.js

```diff
@@ -24,6 +24,7 @@ module.exports = {
     "react/jsx-curly-brace-presence": "off",
     "import/no-extraneous-dependencies": "off",
     "import/prefer-default-export": "off",
+    "react/jsx-wrap-multilines": "off",
```

Comment:
Did we need to turn this off for the whole project?

Reply:
yeah I did because it is a conflict with prettier vs eslint, so prettifies it and then creates this :)